cmd/ig/daemon.go

```diff
@@ -55,6 +56,13 @@ func newDaemonCommand(runtime runtime.Runtime) *cobra.Command {
 		"The socket to listen on for new requests. Can be a unix socket"+
 			" (unix:///path/to.socket) or a tcp socket (tcp://127.0.0.1:1234)")
 
+	daemonCmd.PersistentFlags().Uint64VarP(
+		&size,
```

Comment:
Maybe a more specific name like `eventBufferSize` or `eventBufferLen` is better here.